src/main/java/org/mastodon/views/trackscheme/LineageTreeLayoutImp.java

```diff
@@ -150,10 +156,11 @@ public class LineageTreeLayoutImp implements LineageTreeLayout
 	public LineageTreeLayoutImp(
 			final RootsModel< TrackSchemeVertex > rootsModel,
 			final TrackSchemeGraph< ?, ? > graph,
-			final SelectionModel< TrackSchemeVertex, TrackSchemeEdge > selection )
+			final SelectionModel< TrackSchemeVertex, TrackSchemeEdge > selection, final @Nullable TimepointModel timepointModel )
```

Comment:
Please add a new line after the comma, to keep the rule of one parameter per line.

Reply:
done